backend/go/transcribe/whisper/whisper.go

```diff
@@ -0,0 +1,105 @@
+package main
```

Comment:
This is actually a rename of `backend/go/transcribe/whisper.go` - it's been modified enough that git considers it a new file.

Reply:
transcript.go was also merged into this file to avoid copying locks - it was very whisper specific already